packages/stores/src/providers/ExchangeRatesProvider.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { useEffect } from 'react';
+import { useSubscription } from '@deriv/api';
+import useStore from '../useStore';
+
+const ExchangeRatesProvider = ({ children }: React.PropsWithChildren<unknown>) => {
+    const { data, subscribe } = useSubscription('exchange_rates');
+    const {
+        exchange_rates: { update },
+    } = useStore();
+
+    useEffect(() => {
+        subscribe({ payload: { base_currency: 'USD' } });
+    }, [subscribe]);
+
+    useEffect(() => {
+        if (data) update(data);
```

Comment:
```suggestion
        if (data) update(prev => merge(prev, data));
```
@shayan-deriv Import `merge` from `lodash.merge`